examples/await.rs

```diff
@@ -0,0 +1,52 @@
+#![feature(async_closure)]
+#![feature(proc_macro_hygiene)]
+#![allow(unused_imports)]
+#![allow(unused_variables)]
+#![allow(dead_code)]
+
+use genawaiter::{
+    generator_mut,
+    stack::{Co, Gen, Shelf},
+    yield_,
+};
+
+use gen_proc_macro::{yielder_cls, yielder_fn};
+
+#[yielder_fn(u8)]
+async fn odds() {
+    for n in (1_u8..).step_by(2).take_while(|&n| n < 10) {
+        yield_! { n };
```

Comment:
Can we use `yield_!(n)`, at least in outward-facing code/examples? (parens instead of braces) I don't think the braces are idiomatic Rust.

Reply:
This should be fixed with visit_mut there's 2 to three macro token types.